lib/ansible/modules/packaging/language/pip.py

```diff
@@ -234,6 +252,53 @@
 _SPECIAL_PACKAGE_CHECKERS = {'setuptools': 'import setuptools; print(setuptools.__version__)',
                              'pip': 'import pkg_resources; print(pkg_resources.get_distribution("pip").version)'}
 
+_VCS_RE = re.compile(r'(svn|git|hg|bzr)\+')
+
+op_dict = {">=": operator.ge, "<=": operator.le, ">": operator.gt,
+           "<": operator.lt, "==": operator.eq, "!=": operator.ne, "~=": operator.ge}
```

Comment:
> "~="

I've never heard of it. What is the source of such an assumption?

Reply:
I saw it here https://www.python.org/dev/peps/pep-0440/#version-specifiers, and it's support by `pip`.